src/Renderer/GlobeDepthMaterial.js

```diff
@@ -0,0 +1,44 @@
+
+define('Renderer/GlobeDepthMaterial', ['THREE',
+    'Renderer/BasicMaterial',
+    'Core/System/JavaTools',
+    'Renderer/Shader/GlobeDepthFS.glsl',
+    'Renderer/Shader/GlobeDepthVS.glsl'
+], function(
+    THREE,
+    BasicMaterial,
+    JavaTools,
+    GlobeDepthFS,
+    GlobeDepthVS) {
+
+    var GlobeDepthMaterial = function(otherMaterial) {
+
+        BasicMaterial.call(this);
+
+        this.vertexShader =  GlobeDepthVS;
+        this.fragmentShader = GlobeDepthFS;
+
+        // Why connect directily uniform doesn't work?
+        // Verify attributes's shaders
+
+        this.uniforms.dTextures_00 = {
+            type: "tv",
+            value: otherMaterial.Textures[0]
+        };
+
+        this.uniforms.nbTextures = {
+            type: "iv1",
```

Comment:
You should use a single `int` (elevation texture count) instead of an array of int (elevation + color texture count)